pkg/controller/styra.go

```diff
@@ -22,22 +22,22 @@ import (
 
 	"github.com/crossplane/crossplane-runtime/pkg/logging"
 
-	"github.com/crossplane-contrib/provider-styra/pkg/controller/config"
-	"github.com/crossplane-contrib/provider-styra/pkg/controller/datasource"
-	"github.com/crossplane-contrib/provider-styra/pkg/controller/secret"
-	"github.com/crossplane-contrib/provider-styra/pkg/controller/stack"
+	// "github.com/crossplane-contrib/provider-styra/pkg/controller/config"
+	// "github.com/crossplane-contrib/provider-styra/pkg/controller/datasource"
+	// "github.com/crossplane-contrib/provider-styra/pkg/controller/secret"
+	// "github.com/crossplane-contrib/provider-styra/pkg/controller/stack"
```

Comment:
```suggestion
```
Doesnt go complain about all the comments?

Reply:
These changes were made by mistake.